test/contrib/test_optax.py

```diff
@@ -0,0 +1,106 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+from numpy.testing import assert_allclose
+import pytest
+
+from jax import grad, jit, partial, random
+from jax.lax import fori_loop
+import jax.numpy as jnp
+from jax.test_util import check_close
+import optax
+
+import numpyro
+from numpyro.contrib.optax import optax_to_numpyro
```

Comment:
Could you move imports `optax, optax_to_numpyro` to the content of the tests, like [this](https://github.com/pyro-ppl/numpyro/blob/673880c8c2c50f1bc291af9115fe46e8876554d5/test/contrib/test_tfp.py#L23). This will be helpful when collecting tests for those who do not have `optax`. A better solution is to add
```
try:
    import optax
    from numpyro.contrib.optim import optax_to_numpyro
except ImportError:
    pytestmark = pytest.mark.skip()
```